### PR TITLE
Fix hover effect in volunteer page #504

### LINKED ISSUE #504: fix hover effect in volunteer page
In volunteer page due to hover effect text is also not visible.
@ramyaragupathy  i am working on it.
button

### PR DESCRIPTION
**Pull request by Muskan284**
fixes #504 
@ramyaragupathy  kindly review this pr.

### CODE REVIEW

File: volunteer.markdown [hunk 1]
```diff
@@ -17,7 +17,7 @@ Join the conversation on [Slack](http://slack.hotosm.org/)! A large number of vo
 
 As a mapping volunteer, you can get involved with projects around the world, and contribute to humanitarian mapping through OpenStreetMap. Even if you are a new mapper, anyone can use a computer to create basemaps by drawing over buildings and roads, or use a phone to swipe and tap population hotspots in the MapSwipe app. If you have never mapped before and would like to help, visit the Tasking Manager's [Learn page](https://tasks.hotosm.org/learn) to get started mapping in less than 30 minutes.
 
-<a href="https://tasks.hotosm.org/learn" class="btn btn-primary btn-lg btn-chevron">Learn how to map</a>
+<a href="https://tasks.hotosm.org/learn" class="btn btn-hover btn-primary btn-lg btn-chevron">Learn how to map</a>
```
Comment (harshvardhana25) [inline volunteer.markdown]: If we change this with :- <a href="https://tasks.hotosm.org/learn" class="btn btn-hover btn-primary btn-lg btn-chevron"><color : white>Learn how to map</color></a>

File: _sass/base.scss [hunk 2]
```diff
@@ -621,6 +621,11 @@ em {
     display: block;
     margin-right: 0;
   }
+  &.btn-hover{
+    &:hover{
+      color:$white;
+    }
```
Comment (jayshree1602) [inline _sass/base.scss]: At line no 436

 &.btn-primary {
    color: $white;
    background-color: $red;
    &:hover {
color: white;
      background-color: shade($red, 15%)
    }

**Review by ramyaragupathy — CHANGES_REQUESTED** [review]
Per code [here](https://github.com/hotosm/hotosm-website/blob/gh-pages/_sass/base.scss#L184-L189), hrefs as a child under `<p>` defaults to red color on hover. So instead of creating a new class under `Buttons` section, you could add additional condition to the referred block and control the behaviour.

**Comment by Muskan284** [comment]
@ramyaragupathy I have removed the extra class i created earlier. Kindly review this .

**Comment by Muskan284** [comment]
@ramyaragupathy Kindly review this updated pr.

**Comment by Muskan284** [comment]
@ramyaragupathy Kindly review this updated pr.

**Review by ramyaragupathy — APPROVED** [review]

**Pull request merged**